expyfun/stimuli/tests/test_tracker.py

```diff
@@ -164,12 +163,11 @@ def test_tracker_dealer():
     assert_raises(RuntimeError, dealer_ud.get_trial)
 
     # test array-like indexing
-    dealer_ud[0]
-    dealer_ud[:]
-    dealer_ud[[1, 0, 1]]
-    [d for d in dealer_ud]
+    dealer_ud.trackers[0]
+    dealer_ud.trackers[:]
+    dealer_ud.trackers[[1, 0, 1]]
+    [d for d in dealer_ud.trackers]
     dealer_ud.shape
```

Comment:
Good point. One could get that by doing ``dealer_ud.trackers.shape``. However, since the dealer passes back the subscript for each trial, its shape might be an important enough property that we give people very easy access to it. So I am +0.5 for keeping as is, but I defer to anyone with a strong opinion.